src/test/java/seedu/address/testutil/TypicalWorkers.java

```diff
@@ -8,10 +11,10 @@
 public class TypicalWorkers {
 
     public static final Worker ALICE = new WorkerBuilder().withName("Alice Pauline")
```

Comment:
Can we change ALICE to some other name? Or create another female worker? Amber is already using ALICE and for  `DeleteCommand` I need to test for both `Body` and `Worker`. So I cannot have 2 ALICES :( 
In my `Morph-Delete-Command` branch, I have added another worker